queries/schedule.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This script will create or update BigQuery scheduled queries for each
+# of the derived tables.
+
+# Usage: 
+# `schedule.sh --query_file=<filename_of_source_query> \
+#  --table=<target_table> --project_id=<GCP_project_containing_dataset>`
+
+help() {
+    printf "Usage: schedule.sh --query_file=<filename_of_source_query> --table=<target_table> --project_id=<GCP_project_containing_dataset>\n"
+}
+
+# PARSE INPUTS
```

Comment:
It wasn't picking up my args, so I simplified the parsing a little bit.

Reply:
> Do we want to delete the python script?

Ah, yes --thanks! Removed it.